ZapVersions-2.8.xml

```diff
@@ -429,20 +449,23 @@
         <name>DOM XSS Active scanner rule</name>
         <description>DOM XSS Active scanner rule</description>
         <author>ZAP Dev Team</author>
-        <version>7</version>
-        <file>domxss-alpha-7.zap</file>
+        <version>8</version>
+        <file>domxss-alpha-8.zap</file>
         <status>alpha</status>
-        <changes>Issue 2918: Added an option to attack URL parameters.&lt;br&gt;</changes>
-        <url>https://github.com/zaproxy/zap-extensions/releases/download/2.7/domxss-alpha-7.zap</url>
-        <hash>SHA1:e6d28e52892e41469aefdad3959057900aed6d35</hash>
-        <date>2018-03-07</date>
-        <size>211983</size>
+        <changes>&lt;h3&gt;Changed&lt;/h3&gt;
+&lt;ul&gt;
+&lt;li&gt;Run with Firefox headless by default (Issue 3866).&lt;/li&gt;
+&lt;li&gt;Depend on newer version of Selenium add-on.&lt;/li&gt;
+&lt;/ul&gt;</changes>
+        <url>https://github.com/zaproxy/zap-extensions/releases/download/domxss-v8/domxss-alpha-8.zap</url>
+        <hash>SHA-256:9319100fbff037de500648c74147b4cb58b2d6dffd4f8558b0dcefd269f98679</hash>
+        <date>2019-06-07</date>
+        <size>213687</size>
         <not-before-version>2.7.0</not-before-version>
         <dependencies>
             <addons>
                 <addon>
                     <id>selenium</id>
-                    <semver>2.*</semver>
```

Comment:
This is wrong, the generator is not including the version element (will fix and udpate).

Reply:
This has been fixed.